### ISSUE TITLE
[Request/Proposal] Would it be possible to embed images as Base64 encoded PNGs?

### ISSUE THREAD

**Issue by mimshwright** [bernaferrari/FigmaToCode#128]
I see that there is a note about how to export images from this plugin. I understand that images (technically bitmap fills) can be quite complicated to deal with, but maybe embedding them in the exported HTML as Base64 would be a solution?

**Comment by bernaferrari** [comment]
We could try, maybe as a setting. The issue is that it makes the plugin
extremely slow if there are many images. It can even freeze figma.

**Comment by mimshwright** [comment]
Yes. i understand. I think it should probably be an option that you enable with a warning beforehand. In fact, i could imagine an option that shows thumbnails you can check on and off before exporting. 
Might also be possible to export as a zip of all the image files. But Base64 at least would let you keep everything in a single text box / copy paste action.

**Comment by bernaferrari** [comment]
The issue with exporting zip is that people would want nextjs or something
else and there wouldn't be a way to connect multiple screens together I
think. But we surely can do it. It is easy.

**Comment by mimshwright** [comment]
Discussion in #145 is relevant

**Comment by mimshwright** [comment]
Fixed In #181
